### PR TITLE
Combine scipy and onecodex notebooks

### PR DESCRIPTION
**Pull request by polyatail**
- [X] Merges the scipy and onecodex notebook Dockerfiles.
- [X] No longer installs `bwa` or `samtools`
- [X] No longer installs `tex` packages
- [X] Installs custom `ipyvega` modules which allows setting target=_blank
- [X] Set version of `onecodex` library after we issue a release to include `altair` plots
- [X] Test an image of this on prod

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,36 +1,186 @@
...
+    curl \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
 
-USER 1000
+RUN echo "en_US.UTF-8 UTF-8" > /etc/locale.gen && \
+    locale-gen
+
+# Install Tini
+RUN wget --quiet https://github.com/krallin/tini/releases/download/v0.9.0/tini && \
```
Comment (boydgreenfield) [inline Dockerfile]: Can we update to `v0.18.0`? That's the [latest now](https://github.com/krallin/tini/releases).

File: Dockerfile [hunk 2]
```diff
@@ -1,36 +1,186 @@
...
 RUN pip install --upgrade pip
 
-# Jupyter notebook extensions
-RUN pip install jupyter_contrib_nbextensions && \
-    jupyter contrib nbextension install --user && \
+# Numpy must be installed first
+RUN pip install numpy==1.15.4
+
+# Install Python 3 packages
+RUN pip install \
```
Comment (boydgreenfield) [inline Dockerfile]: These are a bunch of very specific, pinned versions that aren't the latest. Let's update anything that we actually care about and try to cut this list down a bit? 

Specific things that'd be good to ship and are _not_ themselves in `onecodex` (or will be removed):
- `matplotlib`
- `seaborn`
- `numba` and `numexpr`
- `cython`
- `bokeh`?
- `biopython` (ugh)
- `scikit-learn` (or maybe we do have that in requirements?)
- `xlrd`
- `weasyprint`

I'd be cool to ditch at least: `beautifulsoup4`, `vincent`, and `sqlalchemy` unless I'm missing something. Easy to add back in down the line.

Comment (boydgreenfield) [reply Dockerfile]: I'm also good removing `skimage`.

File: Dockerfile (hunk 1 above)
Comment (polyatail) [reply Dockerfile]: For sure, I'll check it out.

File: Dockerfile (hunk 2 above)
Comment (boydgreenfield) [reply Dockerfile]: Previously we'd basically mashed up [this Scipy notebook](https://github.com/jupyter/docker-stacks/tree/master/scipy-notebook) and [this R one](https://github.com/jupyter/docker-stacks/tree/master/r-notebook). [This](https://github.com/jupyter/docker-stacks/blob/master/base-notebook/Dockerfile) is the original base notebook.

File: Dockerfile [hunk 3]
```diff
@@ -1,36 +1,186 @@
...
+# See also http://blog.dscpl.com.au/2016/01/roundup-of-docker-issues-when-hosting.html
 
-# Add assets
-RUN mkdir /opt/onecodex/
-ADD install/* /opt/onecodex/
-RUN chmod +x /opt/onecodex/notebook_report.py && \
-    ln -s /opt/onecodex/notebook_report.py /usr/local/bin/notebook_report.py
+FROM python:3.6.6-slim-stretch
+
+MAINTAINER Nick Greenfield <nick@onecodex.com>
```
Comment (boydgreenfield) [inline Dockerfile]: Apparently this is deprecated and we should now use `LABEL`, e.g.,:

```
LABEL maintainer="Nick Greenfield <nick@onecodex.com>"
```

File: Dockerfile (hunk 1 above)
Comment (polyatail) [reply Dockerfile]: Updated, didn't seem to cause any issues.

File: Dockerfile (hunk 2 above)
Comment (polyatail) [reply Dockerfile]: Okey dokey, trimmed the packages down and pinned more recent versions. Users can always do something like `!pip install --user package` if they need something, I guess.

**Pull request merged**